static/scripts/dashboard/dashboardNewCtrl.js

```diff
@@ -2642,7 +2663,7 @@
     }
 
     function removeWorker (worker) {
-      if (worker.isInitialized)
+      if (worker.isInitialized || worker.isFailed)
```

Comment:
this is for: deleting a swarm worker which is failed for the first time (swarm worker is not initialized) should be added to deletedWorkers.